Ginger/Ginger/Actions/ActionEditPages/WebServices/ActWebAPIEditPage.xaml.cs

```diff
@@ -343,48 +349,98 @@ private void SecCollapsed(object sender, RoutedEventArgs e)
         private void BrowseSSLCertificate(object sender, RoutedEventArgs e)
         {
             string SolutionFolder =  WorkSpace.Instance.Solution.Folder.ToUpper();
-            if (CertificatePath.ValueTextBox.Text != null)
+            if (xCertificatePathTextBox.ValueTextBox.Text != null)
             {
                 // replace Absolute file name with relative to solution
-                string FileName = CertificatePath.ValueTextBox.Text.ToUpper();
+                string FileName = xCertificatePathTextBox.ValueTextBox.Text.ToUpper();
                 if (FileName.Contains(SolutionFolder))
                 {
                     FileName = FileName.Replace(SolutionFolder, @"~\");
                 }
 
-                CertificatePath.ValueTextBox.Text = FileName;
+                xCertificatePathTextBox.ValueTextBox.Text = FileName;
 
                 bool ImportFileFlag = false;
                 Boolean.TryParse(mAct.GetInputParamValue(ActWebAPIBase.Fields.ImportCetificateFile), out ImportFileFlag);
                 if (ImportFileFlag)
                 {
-                    //TODO import Certificate File to solution folder
-                    string targetPath = System.IO.Path.Combine(SolutionFolder, @"Documents\WebServices\Certificates");
-                    if (!System.IO.Directory.Exists(targetPath))
-                    {
-                        System.IO.Directory.CreateDirectory(targetPath);
-                    }
+                    ////TODO import Certificate File to solution folder
+                    //string targetPath = System.IO.Path.Combine(SolutionFolder, @"Documents\WebServices\Certificates");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6345816)